package CfgCard;
import "CfgCommon.proto";

message Card {
	required uint32 id = 1;
	required uint32 skill = 2;
	repeated CfgCommon.Attribute attr = 3;
	repeated CfgCommon.Attribute exAttr = 4;
}

message Cards {
	repeated Card cards = 1;
}


message CardGroup {
	required uint32 groupId = 1;
	required uint32 type = 2;
	required uint32 needNum = 3;
	repeated CfgCommon.Attribute attr = 4;
}

message CardGroups {
	repeated CardGroup groups = 1;
}


message Suit {
	required uint32 id = 1;
	repeated uint32 include = 2;
}

message Suits {
	repeated Suit suits = 1;
}

message Star {
	repeated CfgCommon.Attribute attr = 1;
	required uint32 exp = 2;
	required uint32 color = 3;
	required uint32 level = 4;
}

message Stars {
	repeated Star star = 1;
}

